lib/jobba/status.rb

```diff
@@ -111,7 +116,8 @@ def set_progress(at, out_of = nil)
     end
 
     def set_job_name(job_name)
-      raise ArgumentError, "`job_name` must not be blank" if job_name.nil? || job_name.empty?
+      raise ArgumentError, "`job_name` must not be blank", caller \
```

Comment:
When you pass caller to ArgumentError, it's supposed to ommit the `raise` from the backtrace so it starts on whatever called it... I think